charms/worker/k8s/scripts/rules-patches/002_cpu_utilization.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/charms/worker/k8s/src/prometheus_alert_rules/kubernetesControlPlane-prometheusRule.yaml b/charms/worker/k8s/src/prometheus_alert_rules/kubernetesControlPlane-prometheusRule.yaml
+index 129f859..50e94dd 100644
+--- a/charms/worker/k8s/src/prometheus_alert_rules/kubernetesControlPlane-prometheusRule.yaml
++++ b/charms/worker/k8s/src/prometheus_alert_rules/kubernetesControlPlane-prometheusRule.yaml
+@@ -1636,11 +1636,17 @@ groups:
+       ) by (cluster)
+     record: :node_memory_MemAvailable_bytes:sum
+   - expr: |
+-      avg by (cluster, node) (
++      avg by (cluster, instance) (
+         sum without (mode) (
+-          rate(node_cpu_seconds_total{mode!="idle",mode!="iowait",mode!="steal",job="node-exporter"}[5m])
++          label_replace(
++            rate(node_cpu_seconds_total{mode!="idle",mode!="iowait",mode!="steal",job=~".*node-exporter",juju_application=~".*"}[5m]),
++            "cluster", "$1", "juju_model", "(.*)"
++          )
+         )
+       )
++      *
++      on(instance) group_left(node)
++      label_replace(kube_node_info, "instance", "$1", "node", "(.*)")
```

Comment:
I *KNOW* this is the powerful change.  Can we document any of this to some detail for future travelers?